openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -68,9 +68,12 @@ def convert_empty_to_None(value):
     def get_profile_image(self, obj):
         """ Returns metadata about a user's profile image. """
         data = {'has_image': obj.has_profile_image}
+        if obj.has_profile_image:
```

Comment:
I tend to agree with @dan-f. What was the for changing this behavior, @jimabramson?